code/game/objects/items/storage/belt.dm

```diff
@@ -137,14 +137,14 @@
 	new /obj/item/storage/pill_bottle/bicaridine(src)
 	new /obj/item/storage/pill_bottle/kelotane(src)
 	new /obj/item/storage/pill_bottle/tramadol(src)
+	new /obj/item/storage/pill_bottle/tricordrazine(src)
 	new /obj/item/storage/pill_bottle/dylovene(src)
 	new /obj/item/storage/pill_bottle/inaprovaline(src)
 	new /obj/item/storage/pill_bottle/dexalin(src)
 	new /obj/item/reagent_containers/hypospray/autoinjector/peridaxon_plus(src)
 	new /obj/item/reagent_containers/hypospray/autoinjector/quickclotplus(src)
 	new /obj/item/storage/pill_bottle/spaceacillin(src)
 	new /obj/item/storage/pill_bottle/hypervene(src)
-	new /obj/item/healthanalyzer(src)
```

Comment:
I have to admit, this is a bit iffy doing this, but corpsman has health analyzer in essential kit anyway.

Reply:
I'd do the spaceacillin instead, if anything. Honestly, I'd leave it out of the medbelt, but that's your call.

Reply:
In another headspace, I would replace spaceacillin, though infection control is important (and currently infection is at an all time high for some reason).

Reply:
yeah kill spacecilin